src/ctap/storage.rs

```diff
@@ -258,16 +253,21 @@ pub fn global_signature_counter(
     env: &mut impl Env,
     slot_id: usize,
 ) -> Result<u32, Ctap2StatusCode> {
-    let key = check_and_get_key_for_slot(
-        env,
-        slot_id,
-        key::FIRST_GLOBAL_SIGNATURE_COUNTER,
-        key::GLOBAL_SIGNATURE_COUNTER,
```

Comment:
Signature counters change often and should probably live on multiple storage keys.

Reply:
Done